WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -2059,6 +2059,40 @@ class MySiteViewModelTest : BaseUnitTest() {
         assertThat(siteInfoHeaderCard).isTrue
     }
 
+    @Test
+    fun `given tabs enabled + site menu initial screen, when site menu cards + items, then dynamic card exists`() {
```

Comment:
Can we add two more tests to check that the dynamic card does not exist on the non-initial screen?

1. `given tabs enabled + site menu initial screen, when dashboard cards + items, then dynamic card not exists`
2. `given tabs enabled + dashboard initial screen, when site menu cards + items, then dynamic card not exists`

Reply:
@ashiagr - Great idea. Thanks!! in [72edb6c](https://github.com/wordpress-mobile/WordPress-Android/pull/16277/commits/72edb6c381559c9a6c9dfbd992bedb6ba7c22530). I'll merge after CI finishes. :)